### ISSUE TITLE
Proposal: nodemon option

### ISSUE THREAD

**Issue by ldenoue** [tessel/t2-cli#998]
Would watch files changed on the host computer and automatically restart the node app on the Tessel

**Comment by rwaldron** [comment]
I thought about this early on in the development and evolution of `t2 run/push` and ultimately decided to scratch it because it seems more appropriate to use a build/task runner system for such a thing. I would like to see (and would definitely use) a grunt task for "run on save" where I could `grunt tessel-run-on-save` (or whatever) and it would effectively `t2 run <whatever is presently in package.json's "main">`

**Comment by tcr** [comment]
Yes, I would love to see a grunt task (or external node module, `t2-watch`?) that runs a code monitor and re-runs `t2 run` when code is changed / saved.
